test/controllers/managing.officer.controller.spec.ts

```diff
@@ -139,6 +156,72 @@ describe("MANAGING_OFFICER controller", () => {
       expect(resp.status).toEqual(500);
       expect(resp.text).toContain(SERVICE_UNAVAILABLE);
     });
+
+    test("should render resigned on date when in update journey", async () => {
```

Comment:
there's an equivalent test file for the update version of the page that any update tests should go in. In here don't have mention of Update and vice versa.

Update test file here: test/controllers/update/update.managing.officer.controller.spec.ts

Reply:
Right.. that makes sense. I added them here as some of the tests here were broken by the changes, and the fact the controller for registration calls the same util method `getManagingOfficerById`, I figured there'd be tests here to cover that 🤦.

Reply:
Removed 👍